app/helpers/feedex_helper.rb

```diff
@@ -23,4 +23,12 @@ def get_response_total(total_count, results_limited, scopes)
     response_total = "Over #{response_total}" if results_limited
     response_total
   end
+
+  def confirmation_message(total_count)
+    if total_count && total_count > 1000
```

Comment:
Is 1000 records the limit where we want to start warning people that it's a lot of date?

There's something in feedex where it'll only paginate through `<some number>` of records and has a message saying "There are more records, use CSV export to see them all" on all the pages.  I wonder if whatever that number is and the number we use here should be the same number, or at least that the number we use here should be larger than that number.

Reply:
The max number of pages in `support-api` is set to 200, and the page size is 50, so the results are limited to 10.000. You can see the constants [here](https://github.com/alphagov/support-api/blob/master/app/models/anonymous_contact.rb#L54). I've changed the number to 10.000.